vcsutils/webhookparser/testdata/github/tagcreatepayload.json

```diff
@@ -0,0 +1 @@
+2.0%22%2C%22spdx_id%22%3A%22Apache-2.0%22%2C%22url%22%3A%22https%3A%2F%2Fapi.github.com%2Flicenses%2Fapache-2.0%22%2C%22node_id%22%3A%22MDc6TGljZW5zZTI%3D%22%7D%2C%22allow_forking%22%3Atrue%2C%22is_template%22%3Afalse%2C%22web_commit_signoff_required%22%3Afalse%2C%22topics%22%3A%5B%5D%2C%22visibility%22%3A%22public%22%2C%22forks%22%3A0%2C%22open_issues%22%3A0%2C%22watchers%22%3A0%2C%22default_branch%22%3A%22master%22%2C%22stargazers%22%3A0%2C%22master_branch%22%3A%22master%22%7D%2C%22pusher%22%3A%7B%22name%22%3A%22pavelmemory%22%2C%22email%22%3A%22strokovpavelsergeevich%40gmail.com%22%7D%2C%22sender%22%3A%7B%22login%22%3A%22pavelmemory%22%2C%22id%22%3A7716187%2C%22node_id%22%3A%22MDQ6VXNlcjc3MTYxODc%3D%22%2C%22avatar_url%22%3A%22https%3A%2F%2Favatars.githubusercontent.com%2Fu%2F7716187%3Fv%3D4%22%2C%22gravatar_id%22%3A%22%22%2C%22url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%22%2C%22html_url%22%3A%22https%3A%2F%2Fgithub.com%2Fpavelmemory%22%2C%22followers_url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%2Ffollowers%22%2C%22following_url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%2Ffollowing%7B%2Fother_user%7D%22%2C%22gists_url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%2Fgists%7B%2Fgist_id%7D%22%2C%22starred_url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%2Fstarred%7B%2Fowner%7D%7B%2Frepo%7D%22%2C%22subscriptions_url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%2Fsubscriptions%22%2C%22organizations_url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%2Forgs%22%2C%22repos_url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%2Frepos%22%2C%22events_url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%2Fevents%7B%2Fprivacy%7D%22%2C%22received_events_url%22%3A%22https%3A%2F%2Fapi.github.com%2Fusers%2Fpavelmemory%2Freceived_events%22%2C%22type%22%3A%22User%22%2C%22site_admin%22%3Afalse%7D%2C%22created%22%3Atrue%2C%22deleted%22%3Afalse%2C%22forced%22%3Afalse%2C%22base_ref%22%3Anull%2C%22compare%22%3A%22https%3A%2F%2Fgithub.com%2Fpavelmemory%2Fgo-mockhttp%2Fcompare%2Ftag_intg%22%2C%22commits%22%3A%5B%5D%2C%22head_commit%22%3A%7B%22id%22%3A%225f1317136694845cceb328a0e30b232b9935b759%22%2C%22tree_id%22%3A%22bb26a84104023eab3977d0b6e4b0fa492ad9b002%22%2C%22distinct%22%3Atrue%2C%22message%22%3A%22Add+url+query+support+in+the+request+matcher%22%2C%22timestamp%22%3A%222019-10-24T22%3A51%3A16%2B03%3A00%22%2C%22url%22%3A%22https%3A%2F%2Fgithub.com%2Fpavelmemory%2Fgo-mockhttp%2Fcommit%2F5f1317136694845cceb328a0e30b232b9935b759%22%2C%22author%22%3A%7B%22name%22%3A%22Yinon+Avraham%22%2C%22email%22%3A%22yinonavraham%40gmail.com%22%2C%22username%22%3A%22yinonavraham%22%7D%2C%22committer%22%3A%7B%22name%22%3A%22Yinon+Avraham%22%2C%22email%22%3A%22yinonavraham%40gmail.com%22%2C%22username%22%3A%22yinonavraham%22%7D%2C%22added%22%3A%5B%5D%2C%22removed%22%3A%5B%5D%2C%22modified%22%3A%5B%22request_matcher.go%22%2C%22request_matcher_test.go%22%5D%7D%7D
```

Comment:
Why is this not JSON?

Same for `vcsutils/webhookparser/testdata/github/tagdeletepayload.json`

Reply:
The form-url encoded format is used from the beginning that is why I use it as well.